packages/client/components/DemoSummary.tsx

```diff
@@ -11,6 +13,8 @@ const DemoSummary = () => {
   useCanonical('retrospective-demo-summary')
   useEffect(() => {
     return () => {
+      const localAtmosphere = new LocalAtmosphere().getStore().getSource() as MutableRecordSource
```

Comment:
this is just clearing a brand new local atmosphere. you'll probably want to get the one in use via getLocalAtmosphere()

Reply:
It seems to be working even with a brand new local atmosphere. Do you know why that would be?

Reply:
also i changed the code to look like this instead, but i'm not sure if its correct:
`async function clearLocalAtmosphere() {
  const LocalAtmosphere = await getLocalAtmosphere()
    .then((mod) => mod.default)
    .catch()
  const atmosphere = new LocalAtmosphere().getStore().getSource() as MutableRecordSource
  atmosphere.clear()
}`